internal/source/http.go

```diff
@@ -0,0 +1,87 @@
+package source
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/fs"
+	"net/http"
+	"path/filepath"
+	"time"
+
+	"github.com/go-git/go-billy/v5/memfs"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+)
+
+// http is a bundle source that sources bundles from the specified url.
+type HTTP struct {
```

Comment:
Just a reminder to update to use `https` instead of `http` across the implementation.

Reply:
I will.  Thanks!